maltoolbox/model.py

```diff
@@ -864,4 +722,186 @@ def load_from_file(
             serialized_model = load_dict_from_json_file(filename)
         else:
             raise ValueError('Unknown file extension, expected json/yml/yaml')
-        return cls._from_dict(serialized_model, lang_classes_factory)
+        return cls._from_dict(serialized_model, lang_graph)
+
+
+class ModelAsset:
+    def __init__(self, name:str, lg_asset: LanguageGraphAsset):
```

Comment:
Do we want to make by default all our classes hashable so that we don't have to deal with that if ever needed? It will also make explicit what we consider the id of every object type to be.

Reply:
Do you mean implement a `__hash__` method that returns the id? If so, we can do that, but they might not have an id which complicates things potentially.

Reply:
Let's leave this for when hashable model assets are actually needed.